common/display/displayplanemanager.cpp

```diff
@@ -53,11 +48,17 @@ void DisplayPlaneManager::ResizeOverlays() {
     total_overlays_ = overlay_planes_.size();
     if (total_overlays_ > 1) {
       cursor_plane_ = overlay_planes_.back().get();
+      bool needs_cursor_wa = false;
+#ifdef DISABLE_CURSOR_PLANE
+      needs_cursor_wa = overlay_planes_.size() > 3;
```

Comment:
I am worrying if 1 or more plane are reserved, needs_cursor_wa will not be assigned correct here.

Reply:
@Shao-Feng 
Can you see when this DISABLE_CURSOR_PLANE is defined?
can we remove this MACRO

Reply:
It is defined in BoardConfig.mk under device/intel for different board as "BOARD_CURSOR_WA := true/false".
It could be true in future board